pkg/apis/extensions/v1beta1/defaults.go

```diff
@@ -70,6 +70,7 @@ func SetDefaults_PodSecurityPolicySpec(obj *extensionsv1beta1.PodSecurityPolicyS
 		t := true
 		obj.AllowPrivilegeEscalation = &t
 	}
+	obj.RunAsGroup = extensionsv1beta1.RunAsGroupStrategyOptions{Rule: extensionsv1beta1.RunAsGroupStrategyRunAsAny}
```

Comment:
This should only be set if it's not set already.

Reply:
Also, can you update the defaults_test.go for this?

Reply:
done and done